qutebrowser/config/configdata.py

```diff
@@ -1458,6 +1458,9 @@ def data(readonly=False):
 
     ('normal', collections.OrderedDict([
         ('clear-keychain ;; search', ['<Escape>']),
+        ('set-cmd-text -s :search ', ['/']),
+        ('set-cmd-text -s :search --r', ['?']),
+        ('set-cmd-text -s : ', [':']),
```

Comment:
No need for the trailing spaces here, as they'll be stripped off while parsing the command (but the `-s` option to `set-cmd-text` will insert a space)